pkg/lib/cockpit-components-modifications.css

```diff
@@ -0,0 +1,37 @@
+.modifications-table {
+    border: solid 1px #CCC;
+    margin-top: 15px;
+    margin-bottom: 25px;
+    min-width: 65%;
+    width: 100%;
+}
```

Comment:
As this is quite a lot of custom CSS, I propose to sort this PR after the PF4 rework (PR #11987), and then update the CSS accordingly. Perhaps we'll then also need less of it :-)

Reply:
We maybe should introduce some label and give it to all open PRs that touch css (or possibly even html) to know, that those should be checked if it complies with PF4.